src/components/icons/InfoIcon.svelte

```diff
@@ -1,3 +1,11 @@
+<style>
+  svg {
+    display: inline-block;
+    width: $spacing-md;
+    color: $color-dark-gray-50;
```

Comment:
I might use a lighter shade of grey, to make the info icon a little less prominent. We want it to be visible, but not draw too much attention when people are skimming the content. I think the old glean dictionary got the contrast levels about right:

![image](https://user-images.githubusercontent.com/20569/104326926-44356e00-54b8-11eb-8e0b-046ace628daf.png)